gdbinit.py

```diff
@@ -22,6 +23,13 @@
 venv_path = os.environ.get("PWNDBG_VENV_PATH")
 
 
+def check_poetry():
+    if shutil.which("poetry") is None:
+        print("Poetry is not installed or not found in your PATH.")
+        print("Please run './setup.sh' to install Poetry")
+        sys.exit(1)
```

Comment:
```suggestion
        
```

Reply:
Removing this might fix the docker build. But it would ruin the whole point of having a function to check if poetry is installed.